e2e/volumegroupsnapshot_base.go

```diff
@@ -73,19 +73,20 @@ type VolumeGroupSnapshotter interface {
 }
 
 type volumeGroupSnapshotterBase struct {
-	timeout          int
-	framework        *framework.Framework
-	groupclient      *groupsnapclient.GroupsnapshotV1alpha1Client
-	storageClassName string
-	blockPVC         bool
-	totalPVCCount    int
-	namespace        string
+	timeout                 int
+	framework               *framework.Framework
+	groupclient             *groupsnapclient.GroupsnapshotV1alpha1Client
+	storageClassName        string
+	blockPVC                bool
+	totalPVCCount           int
+	additionalSnapshotCount int
```

Comment:
`additionalSnapshotCount` to `additionalVolumeGroupSnapshotCount` or `additionalVGSnapshotCount`.